src/main/java/seedu/address/logic/parser/KeywordParser.java

```diff
@@ -55,5 +55,36 @@ public KeywordParser(String... inputKeywords){
         return words;
     }
 
+    /**
+     * Similar to parse(), but for input string with one keyword
+     * Returns first match with any keyword. Associated substring is everything after the keyword
+     * @param inputString
+     * @return HashMap containing the keyword - associated substring pair
+     */
+    public HashMap<String, String> parseOne(String inputString){
```

Comment:
Renamed. Though I had planned to replace the three with one parse method later on, when I'm more certain it works properly for all these cases.